README.md

```diff
@@ -37,6 +37,8 @@ type TestUser struct {
   BirthDate     time.Time              `json:"birth_date,omitempty" jsonschema:"oneof_required=date"`
   YearOfBirth   string                 `json:"year_of_birth,omitempty" jsonschema:"oneof_required=year"`
   Metadata      interface{}            `json:"metadata,omitempty" jsonschema:"oneof_type=string;array"`
+  IPAddress2    interface{}            `json:"ip_address2,omitempty" jsonschema:"anyof_ref=#/$defs/ipv4;#/$defs/ipv6"`
```

Comment:
I don't think these are important enough to stick in the README. I think the best way forward here is with some custom tests inside the refect_test.go file.